Packs/SymantecCloudSOC/Integrations/SymantecCloudSOC/SymantecCloudSOC.py

```diff
@@ -0,0 +1,434 @@
+            demisto.debug(f'SymantecEventCollector: Get events request params: {params} ;')
+            response = self._http_request(method='GET', headers=self._headers, params=params)
+        return response.get("logs", []), response.get("next_url")
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """
+    Tests API connectivity and authentication'
+    When 'ok' is returned it indicates the integration works like it is supposed to and connection to the service is
+    successful.
+    Raises exceptions if something goes wrong.
+
+    Args:
+        client (Client): HelloWorld client to use.
+        params (dict): Integration parameters.
+        first_fetch_time (int): The first fetch time as configured in the integration params.
+
+    Returns:
+        str: 'ok' if test passed, anything else will raise an exception and will fail the test.
+    """
+    try:
+        for log_type, url_params in LOG_TYPES.items():
+            demisto.debug(f'test_module: {log_type}')
+            app = url_params.get("app")
+            # created_timestamp is mandatory for investigate.
+            if app == 'Investigate':
+                client.get_events_request(max_fetch=1, app=app,
+                                          subtype=url_params.get("subtype"),
+                                          created_timestamp=datetime.now(timezone.utc).strftime(DATE_FORMAT))
+            else:
+                client.get_events_request(max_fetch=1, app=app, subtype=url_params.get("subtype"))
+
+    except Exception as e:
+        demisto.debug(f'SymantecEventCollector: {str(e)}')
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+
+    return 'ok'
+
+
+def get_events_command(client: Client, first_fetch_time: str,
+                       first_fetch_time_investigate: str,
+                       last_run: dict[str, dict],
+                       limit: int = MAX_LIMIT_PER_CALL) -> tuple[list[dict[str, Any]], CommandResults]:
+    """
+    Gets all the events Symantec API for each log type.
+    Args:
+        client (Client): Symantec Cloud SOC client to use.
+        first_fetch_time (str): The timestamp on when to start fetching events.
+        first_fetch_time_investigate (str): The timestamp on when to start fetching events for investigate.
+        limit (int): The limit of the results to return per log_type.
+        last_run (dict): A dict with a key containing the latest event time we got from last fetch.
+    Returns:
+        list: A list containing the events
+        CommandResults: A CommandResults object that contains the events in a table format.
+    """
+
+    events: list[dict] = []
+    hr = ""
+    for log_type in LOG_TYPES.keys():
+        header_log_type = string_to_table_header(log_type)
+        log_events, _ = get_all_events_for_log_type(
```

Comment:
This code seems to be duplicated with the fetch command was well. 
Better to add the call to another function that will return the events and the time and then call this function from here and also from the fetch